src/app/heading/heading.component.html

```diff
@@ -6,11 +6,11 @@
     <div class="navigation__background">&nbsp;</div>
     <nav class="navigation__nav">
         <ul class="navigation__list">
-            <li class="navigation__item"><a routerLink="/map" class="navigation__link"><span>01</span>Basement</a></li>
+            <!-- <li class="navigation__item"><a routerLink="/map" class="navigation__link"><span>01</span>Basement</a></li>
```

Comment:
I think we can start replacing a lot of the initial design with design from Material Angular.